lldb/tools/lldb-dap/lldb-dap.cpp

```diff
@@ -655,6 +663,8 @@ void request_attach(const llvm::json::Object &request) {
   g_dap.SetFrameFormat(GetString(arguments, "customFrameFormat"));
   g_dap.SetThreadFormat(GetString(arguments, "customThreadFormat"));
 
+  PrintWelcomeMessage();
```

Comment:
We want this in `request_launch(...)` as well as `request_attach(...)` right?

Reply:
Yes, it's defined in both places